src/components/Text/Text.tsx

```diff
@@ -32,9 +33,11 @@ const Text: VibeComponent<TextProps, HTMLElement> & {
       ...typographyProps
     },
     ref
-  ) => {
+    ) => {
+    const { ignoreHeightOverflow } = useContext(TextContext);
     const overrideEllipsis = ellipsis ?? element !== "p";
     return (
+      <TextContext.Provider value={{ignoreHeightOverflow}}>
```

Comment:
```suggestion
      <TypographyContext.Provider value={{ignoreHeightOverflow: false}}>
```
And to make it explicit let's also provide TypographyContext value in `Heading` component, but there it should be 
`{ignoreHeightOverflow: true}` for now